package com.chainsys.exceptions;

import java.util.Scanner;

public class UserDefinedExceptions extends Exception{

	String name;
	int marks;
	public UserDefinedExceptions(String name,int marks) {
		this.name=name;
		this.marks=marks;
	}
	public void info() {
		
		
		Scanner in=new Scanner(System.in);
		System.out.println("Enter name");
		name=in.next();
		
//		for(int i=0;i<5;i++) {
//			System.out.println("Enter marks");
//			marks[i]=in.nextInt();
//		}
		
		try {
		for(int i=0;i<5;i++) {
			if(marks[i]<35) {
				throw new UserDefinedExceptions("Arun",10) ;
			}
		}
		}
		catch(UserDefinedExceptions ex)
	}
	public static void main(String[] args) {
		
		UserDefinedExceptions ude=new UserDefinedExceptions();
		ude.info();
		
		
		

	}

}
